pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-common/src/main/java/org/apache/pinot/plugin/ingestion/batch/common/SegmentGenerationTaskRunner.java

```diff
@@ -142,7 +143,8 @@ private SegmentNameGenerator getSegmentNameGenerator()
       case FIXED_SEGMENT_NAME_GENERATOR:
```

Comment:
Yes, but we can take that up as part of a new refactoring PR. We need to change some more things mentioned in the issue to avoid duplicate props - https://github.com/apache/pinot/issues/8799